benchmark/settings.gradle.kts

```diff
@@ -0,0 +1,2 @@
+rootProject.name = "benchmark"
```

Comment:
`.gradle/` and `build/` are missing from .gitignore

Reply:
(again, commenting under the assumption that Gradle stays for good... which I hope can be avoided ;) )